ipaserver/install/httpinstance.py

```diff
@@ -319,21 +340,46 @@ def create_cert_db(self):
         os.write(f, hex_str)
         os.close(f)
 
-        ipautil.run([paths.CERTUTIL, "-d", database, "-f", pwd_file, "-N"])
-
-        self.fix_cert_db_perms()
-
-    def fix_cert_db_perms(self):
-        pent = pwd.getpwnam(self.service_user)
+        ipautil.run([paths.CERTUTIL, "-d", nssdir, "-f", pwd_file, "-N"])
 
         for filename in NSS_FILES:
-            nss_path = os.path.join(certs.NSS_DIR, filename)
+            nss_path = os.path.join(nssdir, filename)
             os.chmod(nss_path, 0o640)
             os.chown(nss_path, 0, pent.pw_gid)
             tasks.restore_context(nss_path)
 
+    def request_anon_keytab(self):
+        parent = os.path.dirname(paths.ANON_KEYTAB)
+        if not os.path.exists(parent):
+            os.makedirs(parent, 0o755)  # pylint: disable=old-octal-literal
```

Comment:
The thing is, this is *not* an old octal literal, so I don't see why it would complain about it.